docs/conventions/print.md

```diff
@@ -1,8 +1,42 @@
 # How to handle print css rules in ECL
```

Comment:
can you use this template please? https://github.com/ec-europa/europa-component-library/pull/1075/files#diff-6390ccae540cbffdb4a9337ef2f2b8cd :)